src/firebase.js

```diff
@@ -0,0 +1,32 @@
+import { initializeApp } from 'firebase/app';
+import { getMessaging, getToken } from 'firebase/messaging';
+
+const firebaseConfig = {
+  apiKey: process.env.REACT_APP_FIREBASE_API_KEY,
+  authDomain: process.env.REACT_APP_FIREBASE_API_AUTH_DOMAIN,
+  projectId: process.env.REACT_APP_FIREBASE_API_PROJECT_ID,
+  storageBucket: process.env.REACT_APP_FIREBASE_API_STORAGE_BUCKET,
+  messagingSenderId: process.env.REACT_APP_FIREBASE_API_MESSAGING_SENDER_ID,
+  appId: process.env.REACT_APP_FIREBASE_API_APP_ID,
+};
+
+initializeApp(firebaseConfig);
+const messaging = getMessaging();
+
+export const requestForToken = () => {
+  return getToken(messaging, {
+    vapidKey: process.env.REACT_APP_FIREBASE_API_VAPID_KEY_HERE,
+  })
+    .then((currentToken) => {
+      if (currentToken) {
+        console.log('current token for client: ', currentToken);
+      } else {
+        console.log(
+          'No registration token available. Request permission to generate one.',
+        );
+      }
+    })
+    .catch((err) => {
+      console.log('An error occurred while retrieving token. ', err);
+    });
+};
```

Comment:
```suggestion
export const requestForToken = () => {
  try {
    cosnt token =  getToken(messaging, {
      vapidKey: process.env.REACT_APP_FIREBASE_API_VAPID_KEY_HERE,
    });
  
    if (token) {
        console.log('current token for client: ', currentToken);
    } else {
        console.log(
          'No registration token available. Request permission to generate one.',
        );
      }
    })
 } catch(error) {
   console.log('An error occurred while retrieving token. ', err);
 }
};
```

promise를 다음과 같이 async await로 고치시는게 좋아보입니다!